sql-plugin/src/main/scala/com/nvidia/spark/rapids/window/GpuRunningWindowExec.scala

```diff
@@ -75,15 +70,12 @@ class GpuRunningWindowIterator(
     lastOrder = newLastOrder
   }
 
-  def close(): Unit = {
-    if (!isClosed) {
-      isClosed = true
-      fixerIndexMap.values.foreach(_.close())
-      saveLastParts(Array.empty)
-      saveLastOrder(Array.empty)
-      cachedBatch.foreach(_.close())
-      cachedBatch = None
-    }
+  override def doClose(): Unit = {
+    fixerIndexMap.values.foreach(_.close())
```

Comment:
should we fix `foreach(_.close())` in this file to call safeClose in this PR or make it a standalone PR?

Reply:
Since premerge is blocked, moving this to a followup.